composer.json

```diff
@@ -18,7 +18,7 @@
     },
     "require-dev": {
         "ext-intl": "*",
-        "symfony/phpunit-bridge": "^5.0",
+        "symfony/phpunit-bridge": "*",
```

Comment:
Do not make this a wildcard! `^5.1` is fine.

Reply:
To be honest: I'm fine with both. Whats your reason of changing it back to a more fixed version?

Reply:
Wildcards are almost always a bad idea, and lower versions don't work with PHP 8.

Reply:
Ah, I guess what you mean is, that this package (like all symfony packages) has a flexible way of allowing php versions (`>=5.5.9`) and older versions have that too, which could end up in installing an old version of this package with existing or known php 8 issues. I get your worries then. I will change it back to `^5.1`.